Patch by MSP-Greg
Enable native fiber in configure.ac, requires ruby-test_enumerator.rb.patch
diff --git a/configure.ac b/configure.ac
index 5a221f6bf7..25e779be8d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2345,13 +2345,13 @@ AS_CASE(["$target_cpu-$target_os"],
     AC_DEFINE_UNQUOTED(FIBER_USE_COROUTINE, ["$COROUTINE_H"])
     AC_LIBOBJ([coroutine/x86/Context])
   ],
-  # TODO: Enable this after AppVeyor msys2 build succeeds
-  # [x64-mingw32], [
-  #   AC_MSG_RESULT(win64)
-  #   COROUTINE_H=coroutine/win64/Context.h
-  #   AC_DEFINE_UNQUOTED(FIBER_USE_COROUTINE, ["$COROUTINE_H"])
-  #   AC_LIBOBJ([coroutine/win64/Context])
-  # ],
+  # ENABLED FOR ruby-loco TODO: Enable this after AppVeyor msys2 build succeeds
+  [x64-mingw32], [
+    AC_MSG_RESULT(win64)
+    COROUTINE_H=coroutine/win64/Context.h
+    AC_DEFINE_UNQUOTED(FIBER_USE_COROUTINE, ["$COROUTINE_H"])
+    AC_LIBOBJ([coroutine/win64/Context])
+  ],
   [*], [
     AC_MSG_RESULT(no)
   ]
